Short-Answer/Algorithms_Answers.md

```diff
@@ -1 +1,18 @@
-Add your answers to the Algorithms exercises here.
\ No newline at end of file
+#Add your answers to the Algorithms exercises here.
+
+`a)` _O(n)_ - The amount of times the while loop would dependent on n (which is linear) and O(n) is the dominate term.
+
+`b)` _O(n^4)_ - The algorithm has 3 nested for loops inside of a for loop. Each for loop has a time complexity of O(n) and since each of them are dependent on the one that comes before, the time the algorithm runs is n * n * n * n.
```

Comment:
Oooh, almost. Good try, remember that the last loop is constant; it runs 9 times, no matter what!